great_expectations/init_notebooks/sql/validation_playground.ipynb

```diff
@@ -0,0 +1,295 @@
+    "\n",
+    "We'd love it if you **reach out for help on** the [**Great Expectations Slack Channel**](https://greatexpectations.io/slack)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import great_expectations as ge\n",
+    "from great_expectations.profile import ColumnsExistProfiler\n",
+    "import great_expectations.jupyter_ux\n",
+    "from great_expectations.datasource.types import BatchKwargs\n",
+    "from datetime import datetime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Get a DataContext\n",
+    "This represents your project that you just created using `great_expectations init`. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#get-datacontext-object)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "context = ge.data_context.DataContext()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. List the tables in your database.\n",
+    "\n",
+    "The `DataContext` will now introspect your database (a `Datasource`) and list the tables. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#data-assets)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ge.jupyter_ux.list_available_data_asset_names(context)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Pick a table & expectation suite name\n",
+    "\n",
+    "We recommend naming your first expectation suite for a table `warning`. Later, as you identify some of the expectations that you add to this suite as critical, you can move these expectations into another suite and call it `failure`. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/pipeline_integration.html?utm_source=notebook&utm_medium=integrate_validation#choose-data-asset-and-expectation-suite)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "table_name = \"YOUR_TABLE\" # TODO: replace with your value!\n",
+    "expectation_suite_name = \"warning\" # TODO: replace with your value!\n",
+    "data_asset_name = context.normalize_data_asset_name(table_name)\n",
+    "\n",
+    "# Note in a real pipeline you wouldn't create expectations - but we will need some to play with validation.\n",
```

Comment:
If I went through the create expectations notebook just before going here, this is a bit confusing - maybe add some text about running it only if you don't have an expectation suite?

Reply:
Yes, I was really torn on this too. On the one hand I want this to be a completely standalone tutorial that will work. On the other hand if the person has an existing suite they already made it'd be cool if this picked it up. However, given my currently changing philosophy on what makes great tutorials I might leave it as is with a tiny bit of explanatory text.